src/main/java/org/alliancegenome/curation_api/model/document/es/AffectedGenomicModelDocument.java

```diff
@@ -22,7 +22,7 @@ public class AffectedGenomicModelDocument extends ESDocument {
 	private AffectedGenomicModel model;
 	private List<String> associatedPhenotype = new ArrayList<>();
 
-	private Set<DOTerm> diseaseTerms;
+	private Set<DOTerm> diseaseTerms = new HashSet<>();
```

Comment:
Typically we do this when we need the hash set so that the document doesn't get a "null field".